docs/core/tutorials/testing-library-with-visual-studio.md

```diff
@@ -106,31 +99,16 @@ You'll define three methods, each of which calls its <xref:Microsoft.VisualStudi
 
```

Comment:
Suggestion for line 98:

Define three methods, each of which calls an <xref:Microsoft.VisualStudio.TestTools.UnitTesting.Assert> class method repeatedly...